index.css

```diff
@@ -0,0 +1,216 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+}
+
+html,
+body {
+    height: 100%;
+    font-size: 14px;
+    font-family: arial, sans-serif;
+}
+
+.main {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+    background-color: white;
+    position: relative
+}
+
+.navbar {
+    /* background-color: #333; */
+    display: flex;
+    flex-direction: row-reverse;
+    padding-top: 20px;
+    height: 8%
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    align-items: center;
+}
+
+li {
+    display: inline;
+    margin-right: 20px;
+}
+
+/* Style the links */
+a {
+    text-decoration: none;
+    color: black
+}
+
+img {
+    height: 24px;
+}
+
+.title {
+    height: 25%;
+    width: 100%;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: end;
+
+}
+
+.title img {
+    height: 50%;
+
```

Comment:
unnecessary extra line, search about prettier extension in vs code and use that for formatting so that you don't have to take care of such issues.